types/src/connectors/admin/cli.ts

```diff
@@ -112,7 +112,7 @@ export const IntercomCommandSchema = t.type({
   ]),
   args: t.type({
     connectorId: t.number,
-    conversationId: t.union([t.string, t.undefined]),
+    conversationId: t.union([t.number, t.undefined]),
```

Comment:
but conversationId should be a string sId right?

Reply:
oh or is it the conversationId of Intercom not ours? good then

Reply:
yes!